pymongo/database.py

```diff
@@ -680,20 +687,56 @@ def watch(
             show_expanded_events=show_expanded_events,
         )
 
+    @overload
+    def _command(
+        self,
+        sock_info: SocketInfo,
+        command: Union[str, MutableMapping[str, Any]],
+        value: int = 1,
+        check: bool = True,
+        allowable_errors: Optional[Sequence[Union[str, int]]] = None,
+        read_preference: Optional[_ServerMode] = ReadPreference.PRIMARY,
+        codec_options: CodecOptions[Mapping[str, Any]] = DEFAULT_CODEC_OPTIONS,
```

Comment:
How is this different from `codec_options: CodecOptions[_CodecDocumentType] = ...,` below? Should this be `codec_options: None`?

Reply:
I'm confused because later in database.command we use `codec_options: None = None,` in the first overload. Shouldn't we do the same here?

Reply:
In database.command, codec_options can be `None` and if it is none, it'll pass in `DEFAULT_CODEC_OPTIONS` into `_command`. The default value for `codec_options` in `_command` is also not `None` so I don't think it would make sense to have an overloaded version take in `None`

Reply:
I see, in that case this LGTM.